### DISCUSSION TITLE
How am I supposed to know a installed node came from?

### DISCUSSION THREAD

**Discussion by cheezecrisp** [comfyanonymous/ComfyUI#2885] [Q&A] [answered]
The most confusing thing is when I use the search bar and find a node that looks like the one I need, but I don't know exactly how to use it, and I want to go to the author's github page to check the instructions, and I have no idea where the node came from. 

Some authors add specific characters to their node's name to make it immediately obvious where it came from, (like rgthree), but there are also many nodes whose name alone doesn't tell you where they came from.

**Comment by blepping** [comment, marked as answer]
try setting `Badge: Nickname (hide built-in)` in the ComfyUI Manager. then you'll get a little badge with the nickname on top of your nodes:

![image](https://github.com/comfyanonymous/ComfyUI/assets/157360029/9222c5a9-ba81-439b-8eba-2ac309e7a7b6)

that should be enough to find it in the `custom_nodes` directory. if you want to find the exact repo, you can go there and run `git remote -v` which should show you the repo link.

obviously that's not really a streamlined process but at least there is _a_ way, even if it's not very convenient.

**Reply by cheezecrisp** [reply]
Thank You! it's very helpful.
